README.md

```diff
@@ -33,6 +34,9 @@ $ composer require anzusystems/contracts
 
 * [AnzuApp](src/AnzuApp.php)
 * [AnzuUser](src/Entity/AnzuUser.php)
+    * [UserPermissionResolver](src/Security/UserPermissionResolver.php)
```

Comment:
spacing?

Reply:
This is intended, since the resolver is required by the AnzuUser to resolve permissions. I wanted to make that association more apparent.